CONTRIBUTING.md

```diff
@@ -85,6 +85,39 @@ During analysis, each function will be processed into a control flow graph to re
 - run ` ./scripts/setup.sh`
 - After the setup script runs successfully, you should be able to `cd` into `source` and run `make` to build the Pyre binary
 
+### Docker Setup
+
+If you're having issues setting up or your OS is not yet supported, you can also use a Docker image. It runs [Debian GNU/Linux 10 (buster)](https://www.debian.org/) and builds pyre-check from source. 
```

Comment:
Let's not duplicate docs like this. Instead, let's put a pointer here to the other docs. That helps users find them, without adding extra maintenance overhead